docs/automations/extract-bounding-box-metadata.md

```diff
@@ -0,0 +1,141 @@
+| [Mean Relative Area](#mean-relative-area)                                         | The mean of the relative area of bounding boxes in the group                                         | Bounding Box Group Property |
+| [Mean Relative Distance To Image Centre](#mean-relative-distance-to-image-centre) | The mean of the relative distance between each of the bounding box and the image center in the group | Bounding Box Group Property |
+| [Std Aspect Ratio](#std-aspect-ratio)                                             | The standard deviation of the aspect ratios of a group of bounding boxes                             | Bounding Box Group Property |
+| [Std Distance Between Bboxes](#std-distance-between-bboxes)                      | The standard deviation  of the Euclidean distance between each pair of bounding box in the group     | Bounding Box Group Property |
+| [Std Relative Area](#std-relative-area)                                          | The standard deviation of the relative area of bounding boxes in the group                                         | Bounding Box Group Property |
+| [Std Relative Distance To Image Centre](#std-relative-distance-to-image-centre)  | The standard deviation of the relative distance between each of the bounding box and the image center in the group | Bounding Box Group Property |
+
+## Feature Descriptions
+
+### Aspect Ratio
+
+**Aspect ratio** measures the ratio of the bounding box width to its height. It can be useful in scenarios
+where the bounding box shape or dimensions impact the analysis or model performance.
+
+$$
+\text{Aspect Ratio} = \frac{\text{Bounding Box Width}}{\text{Bounding Box Height}}
+$$
+
+### Relative Area
+
+**Relative Area** measures the ratio of the bounding box area to the area of the image its referenced to.
+It can be useful to identify relatively large or small bounding boxes.
+
+$$
+\text{Relative Area} = \frac{\text{Bounding Box Area}}{\text{Image Area}}
+$$
+
+### Relative Distance To Image Centre
+
+**Relative Distance To Image Centre** measures the distance between the center of the bounding box and center
+of the image it is referencing. It can be used to monitor model performance based on how far the object is from
+the center of the image.
+
+### Object Count
+
+**Object count** is a simple count of bounding boxes in a group of bounding boxes. You can use this value
+to monitor how your model performance changes when there are a few or many objects to detect in an image.
+
+### Object Type Count
+
+**Object type count** counts the number of labels in a list of bounding boxes. You can use this count to
+see bow your models perform when classifying a few or many different objects in a scene.
+
+### Combined Area Ratio
+
+**Combined area ratio** is the sum of area bounding box area ratios (sum(bounding box area/image area)).
```

Comment:
```suggestion
**Combined Area Ratio** is the sum of area bounding box area ratios (sum(bounding box area/image area)).
```

Reply:
The `sum(bounding box area/image area)` blurb should probably also be formatted as an equation (not sure if this works):
```
$$
\text{Combined Area Ratio} = \sum{\frac{\text{Bounding Box Area}}{\text{Image Area}}}
$$
```